test/integration/testutils/strings.go

```diff
@@ -45,3 +41,16 @@ func GetLastSplitElement(value string, sep string) string {
 	splitted := strings.Split(value, sep)
 	return splitted[len(splitted)-1]
 }
+
+func EnvNames(branchName string) []string {
+	if branchName == "release-please--branches--main" {
```

Comment:
Is it possible to retrieve `utils.ValFromEnv(t, "TF_VAR_branch_name")` here? Reduce duplication in the code.

Reply:
Are you suggesting change the function to:

```suggestion
	func EnvNames(t *testing.T) []string {
	branchName := utils.ValFromEnv(t, "TF_VAR_branch_name")
	if branchName == "release-please--branches--main" {
		return []string{
			"development",
			"non-production",
			"production",
		}
	}
	return []string{
		"development",
	}
}
```

Reply:
Yeah, not a big deal, but that way only the single call to `EnvNames()` is needed in the various tests.  I'm also fine with the current design if there is a benefit to keep it flexible.